src/validNumberDfa.cpp

```diff
@@ -1,7 +1,7 @@
 #include "states.h"
 #include "validNumberDfa.h"
 
-int error(char c) {
+int validNumberError(char c) {
```

Comment:
If my PR gets merged first, I actually moved error into the base class, so we can all just use that instead.